src/myprofile-page/CreateMutationHelper.js

```diff
@@ -0,0 +1,115 @@
+import { gql } from '@apollo/client';
+import Moment from 'moment';
+import { PLACEHOLDER_MUTATION } from '../sources';
+
+export const createUserEducations = (userId, educations) => {
+  if (!(educations && educations.length > 0)) {
+    return PLACEHOLDER_MUTATION;
+  }
+  const mutations = [];
+  for (let index = 0; index < educations.length; index += 1) {
+    const {
+      institution, description, startDate, endDate,
+    } = educations[index];
+    const mutation = `
+                education${index}: createUserEducation(
+                        userId: ${userId},
+                        institution: "${institution}",
+                        description: "${description}",
+                        startDate: "${Moment(new Date(startDate) * 1000).format('YYYY-MM-DD hh:mm:ss')}",
+                        endDate: "${Moment(new Date(endDate) * 1000).format('YYYY-MM-DD hh:mm:ss')}",
+                    ){
+                    id
+                },
+            `;
+    mutations.push(mutation);
+  }
+
+  return gql`
+        mutation createUserEducation{
+            ${mutations}
+        }
+    `;
+};
+
+export const createUserExperiences = (userId, experiences) => {
```

Comment:
createUserExperiences and createUserEducations - I think by adding an extra argument you may use one function for both cases.

Reply:
fixed, added a `type` which will take part in
`const mutationType = {
     education: 'Education',
     workExperience: 'WorkExperience',
};`